tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.ElevatedServer/DevHome.QuietBackgroundProcesses.QuietBackgroundProcessesManager.idl

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation and Contributors
+// Licensed under the MIT license.
+
+namespace DevHome.QuietBackgroundProcesses
+{
+    [default_interface] runtimeclass QuietBackgroundProcessesManager
```

Comment:
Since the namespace is already "QuietBackgroundProcesses", the class can just be "Manager" or "QBPManager" if you really want to differentiate it.

Reply:
I had that originally, then I noticed that all the APIs keep the full name. 

e.g. Windows.ApplicationModel.AppService has AppServiceConnection class

[https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.appservice.appserviceconnection?view=winrt-22621](https://learn.microsoft.com/en-us/uwp/api/windows.applicationmodel.appservice.appserviceconnection?view=winrt-22621)

I think this is done so that all the using statements don't bring in multiple classes called something generic like "Manager" which then conflict.

personally i agree with your approach though.